autocomplete/complete.go

```diff
@@ -67,7 +68,17 @@ func getPotentialTarget(prefix string) ([]string, error) {
 	}
 	dirPath := splits[0]
 
-	targets, err := earthfile2llb.GetTargets(path.Join(dirPath, "Earthfile"))
+	user, err := user.Current()
+	if err != nil {
+		return nil, err
+	}
+
+	realDirPath := dirPath
+	if strings.HasPrefix(prefix, "~/") {
+		realDirPath = user.HomeDir + "/" + dirPath[2:]
```

Comment:
Should we use `user.HomeDir` in all the places where we've been using `os.Getenv("HOME")`?

Reply:
It's probably a good idea to be consistent. I'll make the switch.